Sources/PackageModel/Manifest.swift

```diff
@@ -69,3 +69,14 @@ extension Manifest {
         return try JSON(string: PackageDescription.jsonString(package: package)).toString(prettyPrint: true)
     }
 }
+
+extension Manifest: Hashable {
```

Comment:
Or perhaps the path, if the manifest has access to that.  But just the name doesn't seem unique enough.